### DISCUSSION TITLE
Activation demande de réservation pour les visiteurs

### DISCUSSION THREAD

**Discussion by yannix63800** [JeromeDevome/GRR#354] [General]
Bonjour à tous, 
Il me semble que tous les visiteurs sont censés pouvoir demander une réservation et ensuite la réservation est modérée
Aujourd'hui, je suis dans une config qui n'autorise que les personnes autorisées à faire des réservations.
Est-ce le fonctionnement normal ou est-ce que j'ai activé une option qu'il ne fallait pas ?
Merci au dev pour ce logiciel génial et merci à tous pour vos conseils.
Yannick

**Comment by ynaessens** [comment]
Bonjour,
cela dépend de la version utilisée. Avec les versions maintenues, cela donne :
- en v3.5.1c, page d'administration, onglet Général, Affichages, choisir le réglage qui vous convient ;
- en v4.3.2, page d'administration, onglet Configuration générale, Fonctionnalités, Formulaire de contact pour réservation, puis choisir le réglage.

Cordialement,
YN

**Comment by yannix63800** [comment]
Tout d'abord, merci pour cette réponse.
Je suis en 4.3.2. J'avais omis d'écrire cette information.
J'ai bien désactivé le formulaire. Mais un utilisateur qui se connecte ne peut pas pour autant demander une réservation.
Les calendriers sont vides (il n'y a pas le + sur la plage horaire).
Je ne sais pas si ce que je souhaite faire est possible.
Je souhaiterais qu'il y ait une modération par domaine ou ressources, car si j'active le formulaire de réservation, c'est pour tous les sites.
Qu'est-ce que j'ai oublié ?
Merci d'avance pour votre retour

**Comment by ynaessens** [comment]
Avez-vous activé la restriction d'accès aux ressources ? (page de paramétrage des ressources)
Vos domaines sont-ils d'accès restreint ? (page de paramétrage des domaines)
Avez-vous mis des délais dans les réservations ? (page de paramétrage des ressources)
J'ai bien les "+" dans mes plannings.

**Comment by yannix63800** [comment]
- Pas de restriction d'accès aux ressources à ma connaissance
![2024-03-19_17-20-18](https://github.com/JeromeDevome/GRR/assets/80893969/df8ff231-0c6d-431d-8779-ffad9ae343b1)
et pourtant sur les ressources appelées C1 du domaine CLER1 : pas de +
![2024-03-19_17-25-01](https://github.com/JeromeDevome/GRR/assets/80893969/465a965c-ca69-4069-be58-1fbb89f95e8e)

- Le domaine n'est pas en accès restreint
![2024-03-19_17-27-03](https://github.com/JeromeDevome/GRR/assets/80893969/c75df9a0-c60e-4dab-8378-08f314563ae9)

- Le délai sont à la valeur par défaut sauf erreur de ma part
![2024-03-19_17-30-04](https://github.com/JeromeDevome/GRR/assets/80893969/98b4539c-0a6f-4c56-a09c-65c40e76dc62)

Merci encore pour le coup de main et vos pistes de recherche

**Comment by ynaessens** [comment]
Une autre idée pour progresser : en administrateur, si vous avez bien accès aux réservations, notez l'adresse de la page d'édition des réservations, puis déconnectez-vous, reconnectez-vous en utilisateur sans droits, copiez l'adresse dans la barre de navigation... la page s'ouvre-t-elle ?

**Comment by yannix63800** [comment]
En administrateur, la page s'ouvre bien
![2024-03-19_17-39-27](https://github.com/JeromeDevome/GRR/assets/80893969/4e03f8ba-7cf9-4853-a019-e971d665f224)
Je vois bien les + mais toujours rien quand je me connecte en utilisateurs sans droits !  :-(
Même en faisant un copier / coller du lien
Là, je ne vois pas du tout quelle bêtise j'ai pu faire pour arriver à cette situation
Une autre idée ?

**Comment by ynaessens** [comment]
Ce n'est pas à la page planning que je pensais, mais celle d'édition des réservations edit_entry.php avec ses paramètres...

**Comment by yannix63800** [comment]
Désolé, je n'avais pas compris.
![2024-03-19_17-49-57](https://github.com/JeromeDevome/GRR/assets/80893969/bb088142-e751-481b-aeb4-46245490edba)
J'ai un joli message "accès refusé"

**Comment by ynaessens** [comment]
Au moins, c'est cohérent avec l'absence des "+".
Reste à trouver pourquoi... vous ne pouvez pas faire une nouvelle installation ?

**Comment by yannix63800** [comment]
Bonjour, 
Réponse un peu tardive.
Les Utilisateurs doivent être USAGER et pas VISITEUR
L'import de mes utilisateurs via active directory considéré mes utilisateurs comme visiteurs.
Du coup problème réglé !
Merci à tous ceux qui m'auront permis de débloquer la situation.
